tasks/27_basic-subclasses/basic-subclasses.js

```diff
@@ -0,0 +1,44 @@
+export class God {
+  static create() {
+    let humans = [];
+    humans.push(new Man('Adam'), new Woman('Eve'));
+    return humans;
+  }
+}
+
+export class Human {
+  constructor(name) {
+    this.name = name;
+    this.sex = undefined;
+  }
+
+  get name() {
+    return this._name;
+  }
+
+  set name(value) {
+    this._name = value;
+  }
+
+  get sex() {
+    return this._sex;
+  }
+
+  set sex(value) {
+    return this._sex = value;
+  }
+}
+
+export class Man extends Human {
+  constructor(name) {
+    super(name);
+    this.sex = 'male';
+  }
+}
+
+export class Woman extends Human {
+  constructor(name) {
+    super(name);
+    this.sex = 'female';
+  }
```

Comment:
Pls use actual ES syntax (you don't have to init properties inside constructor now, take a look [here](https://v8.dev/features/class-fields#public-class-fields)).